projects/batfish-client/src/test/java/org/batfish/client/ClientTest.java

```diff
@@ -251,26 +251,10 @@ public void testValidJsonPath() throws IOException {
       Client.validateJsonPath(mapper.readTree(validJsonPath));
    }
 
-// ratul commented out this test
-//     parseParaValue should allow strings like "I am not valid" to parse.
-//     i can't think of what that new function will not parse
-//       TODO: remove test or give it some teeth
-//   // Tests for parseParaValue method
-//   @Test
-//   public void testParseInvalidJsonContent() {
-//      String invalidJsonContent = "I am not valid";
```

Comment:
@ratulm I don't understand -- the java string `"I am not valid"` is not valid JSON, because the JSON string would be `"\"I am not valid\""` in Java.

(This follows from `"5"` being the JSON number 5 not and `"\"5\""` being the JSON string "5", and from `"false"` being the JSON boolean whereas `"\"false\""` is a JSON string of the word false.)